tools/travis/py3k

```diff
@@ -81,19 +83,21 @@ if git diff | grep "^+ " | grep -v '[ ]*for' | grep -v [.]join | grep -v from_it
     echo
     echo "That was a lot of text!  Read this output from the top for more readable details."
     git reset --hard >/dev/null
-    failed=1
+    warn=1
```

Comment:
What's the story with marking this as a warning rather than a failure?  I think this if clause is for cases where `check_dict_six` changed something that wasn't one of the "unnecessary" cases like for loops, in which case we do want to trigger a failure and ask the user to fix it.